src/server/config.ts

```diff
@@ -265,7 +265,7 @@ export const ApiConfig: ApiDataRequestConfig = {
       rejectUnauthorized: false, // NOTE: Risk is assessed and tolerable for now because this concerns a request to a wel known actor (GH), no sensitive data is involved and no JS code is evaluated.
     }),
   },
-  KLACHTEN: {
+  ENABLEU: {
```

Comment:
```suggestion
  ENABLEU_2_SMILE: {
```

Mogelijk gaan we meer enableu verbindingen krijgen en die zijn niet allemaal met smile.